lib/prerequisite.js

```diff
@@ -12,7 +12,7 @@ module.exports = (input, pkg, opts) => {
 	const tasks = [
 		{
 			title: 'Ping npm registry',
-			skip: () => pkg.private || isExternalRegistry,
+			skip: () => pkg.private || isExternalRegistry || !opts.publish,
```

Comment:
It needs to be handled in the other npm prerequisite tasks too.

Reply:
now also done for
* Verify user is authenticated
* Check npm version

prerequisite tasks